### PR TITLE
Task 2.2 complete

### PR DESCRIPTION
**Pull request by Deryck41**

### CODE REVIEW

**Comment by Deryck41** [comment]
[Задания №2](https://trello.com/c/wXm0OpQK/101-%D0%B7%D0%B0%D0%B4%D0%B0%D0%BD%D0%B8%D1%8F-%E2%84%962)

File: Task2-2/Task2-2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+﻿using System;
```
Comment (PaShkadun) [inline Task2-2/Task2-2/Program.cs:1]: It's not a program, it's piece of shit. Absolutely isn't exists the convention.

File: Task2-2/Task2-2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            else
+            {
+                Console.ForegroundColor = ConsoleColor.Red;
+                Console.WriteLine("Invalid input!");
+                Console.ResetColor();
+            }
+
+            Console.ReadKey();
+        }
+
```
Comment (PaShkadun) [inline Task2-2/Task2-2/Program.cs]: Useless whiteline.

File: Task2-2/Task2-2/Program.cs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                    average = thirdNumber;
+                }
+                else if (min == firstNumber && max == thirdNumber || min == thirdNumber && max == firstNumber)
+                {
+                    average = secondNumber;
+                }
+                else
+                {
+                    average = thirdNumber;
+                }
```
Comment (PaShkadun) [inline Task2-2/Task2-2/Program.cs]: When is option first number with?

File: Task2-2/Task2-2/Program.cs [hunk 4]
```diff
@@ -0,0 +1,53 @@
+﻿using System;
+
+namespace Task2_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Enter three numbers through the whitelines.\n\nNumber 1: ");
+            bool firstSuccessInput = int.TryParse(Console.ReadLine(), out int firstNumber);
```
Comment (AnastasiaKorotkevich) [inline Task2-2/Task2-2/Program.cs:10]: В чем смысл этих переменных?

**Pull request merged**